<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SurveyInitDialogBase</class>
 <widget class="QDialog" name="SurveyInitDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>766</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Survey design</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="mSurveyAreaLayerLabel">
       <property name="text">
        <string>Survey area layer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="mSurveyAreaLayerComboBox"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="mBaselineGroupBox">
     <property name="title">
      <string>Baseline</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QComboBox" name="mSurveyBaselineLayerComboBox"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="mBaselineSimplificationToleranceLabel">
        <property name="text">
         <string>Simplification tolerance</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="mStratumIdComboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="mStratumIdLabel">
        <property name="text">
         <string>Stratum id field</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="mBaselineLayerLabel">
        <property name="text">
         <string>Survey baseline layer</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="mBaselineBufferDistanceSpinBox">
        <property name="minimum">
         <double>-1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
        <property name="value">
         <double>-1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="mBaselineBufferDistanceLabel">
        <property name="text">
         <string>Buffer distance</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="mBaselineSimplificationToleranceSpinBox">
        <property name="minimum">
         <double>-1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
        <property name="value">
         <double>-1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QGroupBox" name="mSurveyDesignGroupBox">
     <property name="title">
      <string>Survey design</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="3" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>240</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="mPointSurveyRadioButton">
          <property name="text">
           <string>Point survey</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="mTransectSurveyRadioButton">
          <property name="text">
           <string>Transect survey</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="mMinimumTransectLengthLabel">
        <property name="text">
         <string>Minimum transect length</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="mShareBaselineCheckBox">
          <property name="text">
           <string>Share baseline</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="mMinimumTransectLengthSpinBox">
        <property name="maximum">
         <double>99999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="mCreateSampleButton">
        <property name="text">
         <string>Create sample sites</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QGroupBox" name="mStrataGroupBox">
     <property name="title">
      <string>Strata</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QComboBox" name="mMinimumDistanceAttributeComboBox"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="mNSamplePointsAttribute">
        <property name="text">
         <string>N sample points attribute</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="mStrataIdAttributeLabel">
        <property name="text">
         <string>ID attribute</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="mNSamplePointsComboBox"/>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="mStrataLayerComboBox"/>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="mStrataIdAttributeComboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="mMinDistanceLabel">
        <property name="text">
         <string>Minimum distance attribute</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="mMinDistanceUnitsLabel">
        <property name="text">
         <string>Distance units</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="mMinDistanceUnitsComboBox"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="mStrataLayerLabel">
        <property name="text">
         <string>Strata layer</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SurveyInitDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SurveyInitDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
